config/policy.go

```diff
@@ -20,9 +21,9 @@ type Policy struct {
 }
 
 // NewPolicy return policy config with specified backend preset
-func NewPolicy(backend string) *Policy {
+func NewPolicy() *Policy {
 	return &Policy{
-		Backend: backend,
+		Backend: ceph.BackendName,
```

Comment:
No. The volume needs a jumping-off-point to create the volume from, and 
that Backend is really it, so policy determines backend, volume just 
carries it.

On 15 Apr 2016, at 10:48, Madhav Puri wrote:

>>  	return &Policy{
>> -		Backend: backend,
>> +		Backend: ceph.BackendName,
>
> shouldn't we remove the `Backend` field from policy altogther?
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/224/files/1f76344ba60e4347be4ef23fb96f4abdb23b0978#r59913001